src/meta-schema-v3.js

```diff
@@ -0,0 +1,174 @@
+module.exports = {
```

Comment:
An idea for a separate issue/PR: I think these meta schemas might make more sense as static .json files